src/CakeSlayer/Has.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
```

Comment:
Let's move this extension to default. I belive we will use it a lot of in this project... As well as `FlexibleInstances` and `FlexibleContexts`